drivers/dac/dac_ad559x.c

```diff
@@ -99,6 +113,7 @@ static int dac_ad559x_init(const struct device *dev)
 #define DAC_AD559X_DEFINE(inst)                                                                    \
 	static const struct dac_ad559x_config dac_ad559x_config##inst = {                          \
 		.mfd_dev = DEVICE_DT_GET(DT_INST_PARENT(inst)),                                    \
+		.range_double = DT_INST_PROP_OR(inst, range_double, false),                        \
```

Comment:
Boolean properties don't need to use _OR.

```suggestion
		.range_double = DT_INST_PROP(inst, range_double),                        \
```